src/components/common/AppBars/navBar.js

```diff
@@ -1,7 +1,8 @@
 import React, { Component } from "react";
-import { Link } from "react-router-dom";
-import Logo from "../../../assets/img/quill-drawing-a-line.svg";
+import { connect } from "react-redux";
+import { Link, withRouter } from "react-router-dom";
 import UserIcon from "../../../assets/img/user.jpg";
+import Logo from "../../../assets/img/quill-drawing-a-line.svg";
 
 class Navbar extends Component {
   constructor(props) {
```

Comment:
@niomwungeri-fabrice We don't need the `constructor` with the react's version that we are using

Reply:
Hi @oesukam!
Those aren't my changes but I can change them.
Thanks